index.ts

```diff
@@ -229,45 +259,47 @@ export function bulk ({
  * @param {string} [$0.url]                     optional URL to override ESRI reverseGeocode endpoint
  * @return {Promise}                            A Promise that'll get resolved with reverse geocode result
  */
-export function reverse ({
+export function reverse({
   clientId,
   clientSecret,
   forStorage = false,
   point,
   url
-}: BaseQuery & {
-  forStorage?: boolean,
-  point: any
+}: BaseQuery & { forStorage?: boolean } & {
+  point: LonLatInput
 }): Promise<BaseResponse> {
```

Comment:
The `BaseResponse` is not even correct here as the query part is a lonlat type. Please update the return type.

Reply:
Fixed in cdd90839c90a85888732fb6718964b8ddd8fb1b6